src/extension/logs/wts.log..2019-12-04

```diff
@@ -0,0 +1,2 @@
+[2019-12-05T16:24:37.359] [INFO] default - [ EXTENSION ]  Launched
```

Comment:
I think you added this one by mistake?

Reply:
yes, we should add the logs folder to .gitignore